.travis.yml

```diff
@@ -23,6 +23,7 @@ before_install:
   - composer global require hirak/prestissimo
 
 install:
+  - travis_retry composer install --no-progress --profile --no-scripts --no-suggest
```

Comment:
why ?

Reply:
The dev dependency on `phpunit/phpunit` is [making this crash on PHP 5.6](https://travis-ci.org/phpro/grumphp/jobs/321901219#L555).

```
53.05s$ travis_retry composer update --no-progress --profile --no-scripts --no-suggest $DEPENDENCIES
[12.5MB/0.01s] Loading composer repositories with package information
[13.0MB/0.28s] Updating dependencies (including require-dev)
[342.2MB/16.08s] Your requirements could not be resolved to an installable set of packages.
[342.2MB/16.08s] 
  Problem 1
    - phpunit/phpunit 6.5.5 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.5.4 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.5.3 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.5.2 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.5.1 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.5.0 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.4.4 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 5.7.25 requires sebastian/diff ^1.4.3 -> satisfiable by sebastian/diff[1.4.3].
    - phpunit/phpunit 5.7.26 requires sebastian/diff ^1.4.3 -> satisfiable by sebastian/diff[1.4.3].
    - Conclusion: don't install sebastian/diff 1.4.3
    - Installation request for phpunit/phpunit ^5.7.25|^6.4.4 -> satisfiable by phpunit/phpunit[5.7.25, 5.7.26, 6.4.4, 6.5.0, 6.5.1, 6.5.2, 6.5.3, 6.5.4, 6.5.5].
```

Not that much alternative solutions I can find at the moment, next to:
- composer/composer#6207
- composer/composer#6856

(as mentioned in the commit message)

Other alternative solutions to look into:
- maybe just using "phars" for these test dependencies, so we can lose the dependency
- restrain the components (drop Symfony 2.7/2.8 support) – but not tested
- ...

Reply:
Weird... Alternative solution, instead of the `.travis.yml` file changes:

```diff
diff --git a/composer.json b/composer.json
index 0fd1ed8..4a93f83 100644
--- a/composer.json
+++ b/composer.json
@@ -29,7 +29,7 @@
         "nikic/php-parser": "~2.1",
         "phpspec/phpspec": "^3.2.2",
         "phpspec/prophecy": "^1.6.2",
-        "phpunit/phpunit": "^5.7.25|^6.4.4",
+        "phpunit/phpunit": "^5.7.26|^6.4.4",
         "sebastian/comparator": "^1.2.4",
         "sensiolabs/security-checker": "^4.0",
         "squizlabs/php_codesniffer": "~2.4"
```

For some reason, that also gets rid of that "conflict"...? What do you think, @keradus?
Should I go with bumping `phpunit/phpunit` a little bit, instead?

Reply:
if bumping phpunit helps, do it.
there was an issue with diff package (premature release of it, rollback, conflicts between it and main package), maybe that's the reason

Reply:
Could be the underlying issue, indeed. I don't quit understand why else it would start happening otherwise, out of the blue?

But, I went with simply bumping `phpunit/phpunit` instead. Fixes the errors, too. And is indeed a lot cleaner... ;-)